client/src/pages/FloorPlan.js

```diff
@@ -31,11 +45,17 @@ const Desk = ({ desk, booking }) => {
 					{booking?.name}
 				</text>
 			</g>
-		</>
+		</g>
 	);
 };
 
-const FloorPlan = ({ desks, bookings }) => {
+const FloorPlan = ({
+	desks,
+	bookings,
+	deskId,
```

Comment:
from the perspective of the floorplan, what does "deskId" mean? Can you think of a better name?

Reply:
I decided to pass isDeskSelected prop which is a boolean value.